pkg/cmd/server/origin/admission/plugin_initializer.go

```diff
@@ -144,6 +145,14 @@ func NewPluginInitializer(
 		cloudConfig,
 		restMapper,
 		quotaRegistry)
+	// upstream broke this, so we can't use their mechanism.  We need to get an actual client cert and practically speaking privileged loopback will always have one
```

Comment:
broke what?

Reply:
> broke what?

They wire to the wrong client cert: https://github.com/kubernetes/kubernetes/issues/53828 .  Also, it's built such that using the "normal" constructor produces a plugin initializer that will panic on a "normal" admission chain.

Reply:
That link helps. Next time I read your PR description.